static/js/invite.js

```diff
@@ -65,9 +65,10 @@ exports.initialize = function () {
         },
         success: function (resp, statusText, xhr, form) {
             $('#submit-invitation').button('reset');
-            var additional_s_if_needed = (invitee_emails.val().match(/@/g) || [] ).length > 1 ? 's' : '';
             invitee_emails.val('');
-            invite_status.text('User' + additional_s_if_needed + ' invited successfully.')
+            var user_word = i18n.t('user', {count: (invitee_emails.val().match(/@/g) || [] ).length});
+            user_word = user_word[0].toUpperCase() + user_word.slice(1);
+            invite_status.text(user_word + ' invited successfully.')
```

Comment:
@TomaszKolek, I think we should mark the whole sentence for translation:
```
invite_status.text(i18n.t('User invited successfully.', {'count': (invitee_emails.val().match(/@/g) || [] ).length}));
```

Also, please add the following key-value pairs in `locale/static/en/translations.json`:
```
{
...
"User invited successfully.": "User invited successfully.",
"User invited successfully._plural": "Users invited successfully."
...
}
```

This way you won't have to add any business logic; i18next will handle the plurals on its own.